lib/src/code_modifier.dart

```diff
@@ -9,7 +9,7 @@ abstract class CodeModifier {
 
```

Comment:
Added redundant types.
We will be pulling changes from BertrandBev, so this file should not be edited without need.
Will not merge this.